### DISCUSSION TITLE
Purpose of storing multiple Kindle serial numbers?

### DISCUSSION THREAD

**Discussion by cyraxote** [apprenticeharper/DeDRM_tools#2300] [Q&A] [answered]
I'm not sure I understand the point of having multiple numbers. Is it simply that if I have more than one Kindle device, any books that are imported in Calibre will be readable on any device? Does it matter that one is a Voyage and the other is a Paperwhite Signature? If I get rid of a device, is there any real reason to get rid of that serial number in DeDRM?

Thanks.

**Comment by ElleKayEm** [comment, marked as answer]
Yes. No. And no.

**Reply by cyraxote** [reply]
Appreciate the brevity. :)

**Reply by cyraxote** [reply]
And here's a follow-up question: are books bought at Amazon associated by them with particular serial numbers, say whichever device is your default when you buy the book?

**Reply by ElleKayEm** [reply]
No, each copy of a book will have DRM associated with the Kindle it's downloaded to.  Or if you download from the website, whichever Kindle you choose then.  The encryption keys for E-ink Kindles are derived from their serial numbers.  If you download a book with Kindle for PC/Mac, DeDRM gets that app's key from where it's stored on your computer.
